katportalclient/test/test_client.py

```diff
@@ -1042,13 +1171,34 @@ def test_sensor_history_single_sensor_invalid_times(self):
         # expect no samples
         self.assertTrue(len(samples) == 0)
 
+    @gen_test
+    def test_sensor_history_exception_on_timeout(self):
+        """Test that exception is raised is download exceeds timeout."""
+        history_base_url = self._portal_client.sitemap[
+            'historic_sensor_values']
+        sensor_name = 'anc_mean_wind_speed'
+        publish_messages = [sensor_history_pub_messages_json['init']]
+        publish_messages.extend(sensor_history_pub_messages_json[sensor_name])
+
+        self.mock_http_async_client().fetch.side_effect = self.mock_async_fetcher(
+            valid_response='{"result":"success"}',
+            invalid_response='error',
+            starts_with=history_base_url,
+            contains=sensor_name,
+            publish_raw_messages=publish_messages,
+            client_states=self._portal_client._sensor_history_states)
```

Comment:
The `publish_raw_messages` and `client_states` must be removed - not used for new katstore.

Also remove `self._sensor_history_states = {}` from `client.py` - it is unused.